src/data/common/clients/storage/Namespaces.ts

```diff
@@ -1,4 +1,5 @@
-export const dataStoreNamespace = "d2-reports";
+export const dataStoreNamespace = "d2-autogen-forms";
+export const formerDataStoreNamespace = "d2-autogen-forms";
```

Comment:
For the new namespace we can use the `D2_AUTOGEN_FORMS` attribute under the `Namespaces` object.

```
// Line 7
export const Namespaces = {
    ...
    D2_AUTOGEN_FORMS: "d2-autogen-forms",
};
```